ConsoleProject/src/services/TextureConverter.cs

```diff
@@ -99,7 +153,9 @@ public class TextureConverter {
             System.Runtime.InteropServices.Marshal.Copy(data.Scan0, bytes, 0, bytes.Length);
 
             for (int x = 0; x < texture.Unswizzled.Length; x++) {
-                if (x >= texture.Width + (texture.Width * row)) row++;
+                if (x >= texture.Width + (texture.Width * row)){
```

Comment:
Could you explain this condition? It feels like it can be replaced with modulo arithmetic.
if ( x % texture.Width == 0) { row++; }

Reply:
Indeed, changed to modulo!